<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StageChronicle</class>
 <widget class="QWidget" name="StageChronicle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>阶段记事</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,4">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="4" column="3" rowspan="2" colspan="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>     ，签收时间：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>调度台:   </string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" rowspan="3" colspan="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>发送调度员：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="3" rowspan="3" colspan="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>     ，发送时间：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3" colspan="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>XX台，班别-阶段号：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2" rowspan="3">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0" rowspan="2" colspan="2">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>签收值班员：</string>
              </property>
             </widget>
            </item>
            <item row="4" column="5">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTextBrowser" name="textBrowser"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>签收</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>111</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
